decred/tests/unit/dcr/conftest.py

```diff
@@ -31,3 +31,25 @@ def queue(k, v):
         q.setdefault(k, []).append(v)
 
     return queue
+
+
+@pytest.fixture
+def MockWebSocketClient_class():
+    class MockWebSocketClient:
```

Comment:
Moved here so that it is used by both `test_dcrdata_unit.py` and `test_rpc_unit.py`.